gcc/d/ChangeLog

```diff
@@ -1,3 +1,11 @@
+2018-09-25  Eugene Wissner  <belka@caraus.de>
+
+	* d-builtins.cd (d_eval_constant_expression): Don't make STRING_CSTs
+	larger than they are.
```

Comment:
This entry can be removed (make sure the comment is copied below! :-)

Reply:
An interesting fact is that I did it already, but forgot to stage ChangeLog.